fifteen_min/src/find_home.rs

```diff
@@ -19,8 +19,8 @@ pub struct FindHome {
 }
 
 impl FindHome {
-    pub fn new(ctx: &mut EventCtx, options: Options) -> Box<dyn State<App>> {
-        let panel = Panel::new(Widget::col(vec![
+    pub fn new_state(ctx: &mut EventCtx, options: Options) -> Box<dyn State<App>> {
```

Comment:
@michaelkirk, how do you feel about `new_state` and `new_builder`? I understand the motivation behind this lint in general, but for things like creating a new UI state, this feels unnecessarily noisy. Callers are calling these and wrapping in `Transition::Push` or such, so it's quite clear that it's a new state.

Reply:
I like this change if you're open to it.

In the tradeoff between concise vs. predictable, I tend to favor predictable. i.e. `new` should return `Self`

Reply:
I've warmed up to this change; it indeed makes the return type much more predictable. Looks good!